projects/sbb-esta/angular-business/chip/src/chip/chip.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, EventEmitter, HostBinding, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'sbb-chip',
+  templateUrl: './chip.component.html',
+  styleUrls: ['./chip.component.scss']
+})
+export class ChipComponent {
+  @Input()
+  label: string;
+
+  @Input()
+  disabled = false;
+
+  @Output()
+  dismissed: EventEmitter<boolean> = new EventEmitter();
```

Comment:
What do you think about emitting ChipComponent (`.emit(this)` in dismiss)?
This would allow a consumer to figure out, which chip was dismissed.

Reply:
Sounds good :)